internal/cmd/pipeline/command_update.go

```diff
@@ -31,11 +32,20 @@ func (c *command) newUpdateCommand(enableSourceCode bool) *cobra.Command {
 				Text: `Revoke privilege to activate Stream Designer pipeline "pipe-12345".`,
 				Code: `confluent pipeline update pipe-12345 --activation-privilege false`,
 			},
+			examples.Example{
+				Text: `Update Stream Designer pipeline "pipe-12345" with KSQL cluster ID "lksqlc-123456".`,
+				Code: `confluent pipeline update pipe-12345 --ksql-cluster lksqlc-123456.`,
```

Comment:
Code snippets shouldn't end with periods
```suggestion
				Code: "confluent pipeline update pipe-12345 --ksql-cluster lksqlc-123456",
```